conftest.py

```diff
@@ -8,3 +11,18 @@
     "bigquery_etl.pytest_plugin.script_lint.flake8",
     "bigquery_etl.pytest_plugin.script_lint.mypy",
 ]
+
+
+def pytest_collection_modifyitems(config, items):
+    keywordexpr = config.option.keyword
+    markexpr = config.option.markexpr
+    if keywordexpr or markexpr:
+        return
+
+    skip_publish_json_script = pytest.mark.skip(
+        reason='publish_json_script marker not selected'
+    )
```

Comment:
I decided to add a custom marker to prevent running the tests that test the `publish_public_data_json` script in the CI. To run those tests locally that require access to GCP: `pytest -m publish_json_script`

Reply:
I like the custom marker but don't like that it doesn't run in CI. We could unintentionally run into a situation where tests fail but CI doesn't catch it. Can we add integration testing with proper service accounts in CI?

Reply:
I'm all for it. We could have a GCP project specifically intended for running integration tests that doesn't have access to any actual data. I guess setting up such a project will require ops to do it.

Reply:
I set up a sandbox account for gcp-ingestion's CI tests without ops involvement. We could do the same here, and we can perhaps follow up with ops about whether they want to own these projects instead. We probably don't want to block this work on ops availability to provision the things. Feel free to ask me questions about how gcp-ingestion's CI tests and creds are set up.

Reply:
Sounds good. I'll take a look tomorrow.

Reply:
I set it up for CI.